doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_fota_update.rst

```diff
@@ -0,0 +1,48 @@
+:orphan:
+
+.. _ug_nrf54h20_suit_fota_update:
+
+FOTA updates on nRF54H Series devices
+#####################################
+
+|fota_upgrades_def|
+You can also use FOTA updates to replace the application.
+
+FOTA over Bluetooth Low Energy
+******************************
```

Comment:
```suggestion
FOTA over Bluetooth® Low Energy
*******************************
```

Reply:
this needs to be aligned in our active documentation (see nRF52/53/54L guides)